docs/context/v1.1.0/phyx.json

```diff
@@ -0,0 +1,148 @@
+{
+  "@context": {
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "obo": "http://purl.obolibrary.org/obo/",
+    "dwc": "http://rs.tdwg.org/dwc/terms/",
+    "dct": "http://purl.org/dc/terms/",
+    "tc": "http://rs.tdwg.org/ontology/voc/TaxonConcept#",
+    "tn": "http://rs.tdwg.org/ontology/voc/TaxonName#",
+    "testcase": "http://vocab.phyloref.org/phyloref/testcase.owl#",
+    "pso": "http://purl.org/spar/pso/",
+    "tvc": "http://www.essepuntato.it/2012/04/tvc/",
```

Comment:
There seem to be some prefixes such as this that aren't actually used? I notice they were there before; if there's no use for them, wouldn't it be better to eliminate them?

Reply:
Actually, `pso`, `tvc` and `timeinterval` are used by the [PhylorefWrapper.getCurrentStatus()](https://github.com/phyloref/phyx.js/blob/f705301c8c02fffcf784a64085699872593d9948/src/wrappers/PhylorefWrapper.js#L176), [PhylorefWrapper.getStatusChanges()](https://github.com/phyloref/phyx.js/blob/f705301c8c02fffcf784a64085699872593d9948/src/wrappers/PhylorefWrapper.js#L219) and [PhylorefWrapper.setStatus()](https://github.com/phyloref/phyx.js/blob/f705301c8c02fffcf784a64085699872593d9948/src/wrappers/PhylorefWrapper.js#L253) methods, which were once used to document changes in status in a phyloreference. I think our current intention is to remove this from PhylorefWrapper and to instead use external systems to track phyloref statuses (e.g. GitHub), except for deprecation. But we should probably leave this prefixes in until they are removed from the source code. I'm going to track that in https://github.com/phyloref/phyx.js/issues/135.

`testcase` and `opentree` are actually no longer being used in phyx.js, and so I've removed them in bbc26ee.